packages/arb-ts/integration_test/standarderc20.test.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  providers,
+  utils,
+  Wallet,
+  BigNumber,
+  constants,
+  ethers,
+  ContractReceipt,
+} from 'ethers'
+import { Bridge } from '../src/lib/bridge'
+import { Network } from '../src/lib/networks'
+
+import { expect } from 'chai'
+import config from './config'
+import { TestERC20__factory } from '../src/lib/abi/factories/TestERC20__factory'
+
+import yargs from 'yargs/yargs'
+import chalk from 'chalk'
+import {
+  fundL1,
+  fundL2,
+  testRetryableTicket,
+  prettyLog,
+  warn,
+  instantiateRandomBridge,
+  fundL2Token,
+  tokenFundAmount,
+  skipIfMainnet,
+  existentTestERC20,
+} from './testHelpers'
+const { Zero, AddressZero } = constants
+
+describe('standard ERC20', () => {
+  beforeEach('skipIfMainnet', function () {
+    skipIfMainnet(this)
+  })
+
+  it('deposits erc20 (no L2 Eth funding)', async () => {
+    const { bridge } = await instantiateRandomBridge()
+    await fundL1(bridge)
+    await depositTokenTest(bridge)
+  })
+  it.skip('deposits erc20 (with L2 Eth funding)', async () => {
+    const { bridge } = await instantiateRandomBridge()
+    await fundL1(bridge)
+    await fundL2(bridge)
+    await depositTokenTest(bridge)
+  })
+
+  it('withdraws erc20', async function () {
+    const tokenWithdrawAmount = BigNumber.from(1)
+    const { bridge } = await instantiateRandomBridge()
```

Comment:
should we just instantiate this in the beforeEach hook?

Reply:
prob, but punting